libdnf/base/base.cpp

```diff
@@ -32,6 +32,7 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 #include <atomic>
 #include <cstdlib>
 #include <filesystem>
+#include <functional>
```

Comment:
Sorry, just last thing, can you take this out? We are including it already in the header...

Reply:
Sure!